lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -167,6 +163,8 @@ class CategoryRenderer {
     const titleEl = this._dom.createChildOf(summary, 'div', 'lh-perf-hint__title');
     titleEl.textContent = audit.result.description;
 
+    this._dom.createChildOf(summary, 'div', 'lh-toggle-arrow', {title: 'See resources'});
```

Comment:
This is the item level arrow in Opportunities, which is expandable:
![image](https://user-images.githubusercontent.com/39191/28389571-84a9687e-6c8b-11e7-8922-5ac4acd07121.png)

afaik, all non expandable arrows correctly have that class.